mlrun/runtimes/mpijob/v1.py

```diff
@@ -72,11 +72,15 @@ def _enrich_launcher_configurations(self, launcher_pod_template):
         quoted_args = []
         for arg in self.spec.args:
             quoted_args.append(shlex.quote(arg))
+        quoted_mpi_args = []
+        for arg in self.spec.mpi_args:
+            quoted_mpi_args.append(shlex.quote(arg))
         if self.spec.command:
             self._update_container(
                 launcher_pod_template,
                 "command",
-                ["mpirun", "python", shlex.quote(self.spec.command)] + quoted_args,
+                ["mpirun", *quoted_mpi_args, "python", shlex.quote(self.spec.command)]
+                + quoted_args,
```

Comment:
If you added the mpi args inside, do the same for the args, i.e.:
```suggestion
                ["mpirun", *quoted_mpi_args, "python", shlex.quote(self.spec.command), *quoted_args]
```